src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -59,7 +59,8 @@ See the LICENSE file in the project root for more information.
     <NativeBinaryExt Condition="'$(NativeCodeGen)' == 'wasm'">.html</NativeBinaryExt>
 
     <ExportsFileExt Condition="'$(OutputType)' !=  'Exe' and '$(TargetOS)' == 'Windows_NT' and '$(NativeLib)' == 'Shared'">.def</ExportsFileExt>
```

Comment:
Nit: The conditions on `ExportsFileExt` can be simplified:
```
 <ExportsFileExt Condition="'$(TargetOS)' == 'Windows_NT'">.def</ExportsFileExt>
 <ExportsFileExt Condition="'$(TargetOS)' == 'OSX'">.exports</ExportsFileExt>
...
```

Reply:
ExportsFileExt should only be generated when final binary is a shared library, shouldn't?

Reply:
Right, the exports file should be generated only when final binary is shared library.

But it is simpler to set `ExportsFileExt` property all the time, even when the exports file is not actually generated. It does not hurt for it to be set even when the value is not actually used.